### ISSUE TITLE
[Question] `Visualization` performance

### ISSUE THREAD

**Issue by goekce** [Farama-Foundation/Gymnasium#798]
### Question

I have been playing with https://gymnasium.farama.org/api/utils/#visualization for a while and noticed that running the provided example `CarRacing-v2` can lead to an unplayable game. If I start using `python /usr/lib/python*/site-packages/gymnasium/envs/box2d/car_racing.py` then the gameplay is very fluent.

1. Is this related to the difference between `rgb_array` and `human`? 
2. Why does `Visualization` require `rgb_array`?

**Comment by Kallinteris-Andreas** [comment]
I cannot replicate this issue on my end.
2. There is no particular design reason why the play function only supports `render_mode=rgb_array`

**Comment by goekce** [comment]
> There is no particular design reason why the play function only supports ...

However it is implemented so:

https://github.com/Farama-Foundation/Gymnasium/blob/a79e5d6e8a41885245bea481e36746539ecbf872/gymnasium/utils/play.py#L54-L58

---

I compared again. I find `play` a tiny bit faster than `python /usr/lib/python*/site-packages/gymnasium/envs/box2d/car_racing.py`, so I still have the impression that `play` renders a bit faster. I cannot remember anymore if `play` was much faster when I tested in December. Currently the difference is insignificant, so I will close this question.
